docs/api/instruments/hp/hp3478A.rst

```diff
@@ -0,0 +1,7 @@
+######################################
+HP 3478A Multimeter
+######################################
+
+.. autoclass:: pymeasure.instruments.hp.HP3478A
+    :members:
+    :show-inheritance:
```

Comment:
This file is not part of your this pull request and it is actually part of the master branch.
Again, something wrong with rebase ?

Reply:
yeah, messed up the rebase